framework/core/src/Http/Middleware/AuthenticateWithHeader.php

```diff
@@ -41,7 +41,7 @@ public function process(Request $request, Handler $handler): Response
                 $request = $request->withAttribute('apiKey', $key);
                 $request = $request->withAttribute('bypassThrottling', true);
             } elseif ($token = AccessToken::findValid($id)) {
-                $token->touch($request);
+                $token->touch(null, $request);
```

Comment:
Can't we use:

```suggestion
                $token->touch(request: $request);
```

Or is the php version too low?

Reply:
php 8.1 has named parameters, so we can

Reply:
Yep, I think we can do that from 8.0 actually. Good shout 👍